.eslintrc.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
     'no-debugger': 'off',
     'react/prefer-stateless-function': 'warn',
```

Comment:
Please remove it and use functional component where possible

Reply:
Done